airbrakes/logger.py

```diff
@@ -55,13 +55,6 @@ def start(self):
         """
         self._log_process.start()
 
-    @property
-    def is_running(self) -> bool:
```

Comment:
aw but I liked that

Reply:
we weren't using it anywhere, do you think we should/we will use it?

Reply:
we probably won't, but it's a nice to have to know the logger is actually running.